packages/react-strict-dom/postcss/bundler.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+const babel = require('@babel/core');
+const reactStrictBabelPreset = require('react-strict-dom/babel-preset');
```

Comment:
Replaced StyleX Plugin with RSD Babel Preset to match behaviors exactly as described in docs